python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -45,7 +53,8 @@ def prepare(self, force=False):
         """
         Base class for all applications which can be placed into a prepared\
         state. 
-
+        Args:
+            force (bool) : ignored by the base class. Higher up forces the prepare function to be called no matter what
```

Comment:
As I say if there is a cheat sheet according to some standard for those who care about these details I'm very happy to follow it. But this is a valid statement for this attribute in this class so why not state this?